src/spikeinterface/sorters/tests/test_launcher.py

```diff
@@ -31,19 +41,106 @@ def setup_module():
         rec.save(folder=rec_folder)
 
 
-def test_run_sorters_with_list():
-    working_folder = cache_folder / "test_run_sorters_list"
+def get_job_list():
+    jobs = []
+    for i in range(num_recordings):
+        for sorter_name in sorters:
+            recording = load_extractor(cache_folder / f"toy_rec_{i}")
+            kwargs = dict(
+                sorter_name=sorter_name,
+                recording=recording,
+                output_folder=base_output / f"{sorter_name}_rec{i}",
+                verbose=True,
+                raise_error=False,
+            )
+            jobs.append(kwargs)
+
+    return jobs
+
+
+@pytest.fixture(scope="module")
+def job_list():
+    return get_job_list()
+
+
+def test_run_sorter_jobs_loop(job_list):
+    if base_output.is_dir():
+        shutil.rmtree(base_output)
+    sortings = run_sorter_jobs(job_list, engine="loop", return_output=True)
+    print(sortings)
+
+
+def test_run_sorter_jobs_joblib(job_list):
+    if base_output.is_dir():
+        shutil.rmtree(base_output)
+    sortings = run_sorter_jobs(
+        job_list, engine="joblib", engine_kwargs=dict(n_jobs=2, backend="loky"), return_output=True
+    )
+    print(sortings)
+
+
+def test_run_sorter_jobs_processpoolexecutor(job_list):
+    if base_output.is_dir():
+        shutil.rmtree(base_output)
+    sortings = run_sorter_jobs(
+        job_list, engine="processpoolexecutor", engine_kwargs=dict(max_workers=2), return_output=True
+    )
+    print(sortings)
+
+
+@pytest.mark.skipif(True, reason="This is tested locally")
```

Comment:
dask should not be installed in tests. Why not having the usual `HAVE_DASK ...` and `skipif(not HAVE_DASK, reason="dask is not installed")`?

Reply:
It is a big beast and dask is useless in our case without dask.distributed which also needs some slurm like infrastruture to be tested. So too heavy for me.